tests/fixture/tmpnet/network.go

```diff
@@ -1117,13 +1120,28 @@ func getRPCVersion(log logging.Logger, command string, versionArgs ...string) (u
 // with the given UUID. The start and end times are accepted as strings to support the
 // use of Grafana's time range syntax (e.g. `now`, `now-1h`).
 func MetricsLinkForNetwork(networkUUID string, startTime string, endTime string) string {
+	return NewGrafanaURI(
+		networkUUID,
+		startTime,
+		endTime,
+		GetEnvWithDefault("GRAFANA_URI", defaultGrafanaURI),
+	)
+}
+
+// NewGrafanaURI returns a Grafana dashboard URI.
+func NewGrafanaURI(
```

Comment:
How do the two different names NewGrafanaURI and MetricsLinkForNetwork make it clear what the difference between the two is?